standalone/src/screens/WebcamCaptureTool.tsx

```diff
@@ -113,6 +113,7 @@ export default function WebcamCaptureTool(props: ICommonProps) {
     function renderPageButtons() {
         return (
             <WebcamCaptureToolButtons
+                t={props.t}
```

Comment:
I think the `t` could  a bit tiny as a symbol to use for translations here. Not a blocker for the PR though. How do you feel about hooks here? `useTranslation` instead of `props`

Reply:
Thought about it - reason for using `props` was to enforce language consistently across the app, and if you delegate the responsibility for instantiating the translator to the components, you might get mismatches, faulty setup, etc.

I'll admit that one eye is towards creating a way to switch languages via the UI, so maybe I'm letting that overcomplicate things.

I'm cool with changing it from `t`, but that is the convention for `i18next` translation functions, so what we gain in specificity we lose in familiarity, so to speak.